pack/tic/itm_encounter.json

```diff
@@ -0,0 +1,578 @@
+        "clues": null,
+        "code": "07317",
+        "double_sided": true,
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 7,
+        "faction_code": "mythos",
+        "flavor": "You've agreed to help Agent Harper complete her mission to find and bring back the vast wealth of Y'ha-nthlei. Who knows what the government will do with such riches...",
+        "illustrator": "Carlos Palma Cruchaga",
+        "name": "City of the Deep (v. II)",
+        "pack_code": "itm",
+        "position": 317,
+        "quantity": 1,
+        "stage": 2,
+        "text": "<b>Objective</b> - Recover the riches of the Deep Ones. If the Lair of Dagon, the Lair of Hydra, and the Vault of Riches with the purple key have no clues on them, advance.",
+        "type_code": "act"
+    },
+    {
+        "back_flavor": "With the deities of the Deep Ones thrashing and howling in pain, sections of the city begin to collapse all around you. Hoping to use this to your advantage, you lure the creatures closer and closer to the vaults where their riches are held. The vast hoards of gold and jewels are no doubt how the creatures - and by proxy, the Esoteric Order of Dagon - were able to manipulate the people of Innsmouth and bring more into their fold. You cannot let that ever happen again. One final blow sends one of the creatures staggering through the wall. The ceiling collapses, and a torrent of seawater follows. You escape just before you are caught inside.",
+        "back_name": "Conspiracy Dismantled",
+        "back_text": "Add this card to the victory display.\n- If there is another act card in play, you may choose to either continue playing in order to accomplish the objective on that act card, or proceed immediately to <b>(→R1)</b> without fulfilling that objective.\n- If there are no other act cards in play, proceed to <b>(→R1)</b>.\n<b>Victory 2.</b>",
+        "clues": null,
+        "code": "07318",
+        "double_sided": true,
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 8,
+        "faction_code": "mythos",
+        "flavor": "You refuse to be a pawn in the government's schemes. Even so, you don't wish Agent Harper to come to any harm. You tell her to escape while she can, and she slips away, cursing under her breath. Now you have a new goal - send the riches of Y'ha-nthlei into the depths so they can never again be used against humanity.",
+        "illustrator": "Carlos Palma Cruchaga",
+        "name": "City of the Deep (v. III)",
+        "pack_code": "itm",
+        "position": 318,
+        "quantity": 1,
+        "stage": 2,
+        "text": "<b>Objective</b> - Collapse the vault containing the Deep Ones' riches. If Dagon and Hydra each have at least 4 [per_investigator] damage on them, advance.",
+        "type_code": "act"
```

Comment:
Should these three acts all have  `"victory": 2` on them?  That's usually how we handle victory, but I think this is the first time the back side of an act/agenda has had 'victory' on it.